src/arrayMethodSort.js

```diff
@@ -3,9 +3,35 @@
 /**
  * Implement method Sort
  */
+const compareAsStrings = (a, b) => {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  }
+
+  if (stringA === stringB) {
+    return 0;
+  }
+
+  return -1;
+};
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareAs = compareAsStrings) {
```

Comment:
The parameter name 'compareAs' is a bit misleading as it suggests that it might be a boolean flag. A more conventional name for a comparison function parameter is 'compareFn' or 'comparator'.